### PR TITLE
IOSXE ignore macSecControlledIF and macSecUncontrolledIF ifTypes

### PR DESCRIPTION
**Pull request by pepperoni-pi**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Support/Configuration.md [hunk 1]
```diff
@@ -650,16 +650,34 @@ you don't need to configure full location within snmp.
 
 # Interfaces to be ignored
 
+Interfaces can automatically be ignored during discovery by adding bad_if\* 
+entries in either the OS specific definition (includes/definitions/_specific_os_.yaml)
```
Comment (murrant) [inline doc/Support/Configuration.md]: This info is good to include, but we should not suggest users edit yaml files.  This has a high chance to block updates.

FYI yaml can be overriden in config.php.

Comment (pepperoni-pi) [reply doc/Support/Configuration.md]: Sweet, alright I see that now under [here](https://docs.librenms.org/Developing/os/Settings/#user-override-in-configphp). I'll modify my documentation updates.

Comment (pepperoni-pi) [reply doc/Support/Configuration.md]: I have updated the documentation portion of this pull-request.

**Comment by pepperoni-pi** [comment]
OS specific version of my previous [pull-request] (https://github.com/librenms/librenms/pull/11209).

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-61-release-changelog-february-2020/11145/1

**Pull request merged**